src/styles/card.css

```diff
@@ -0,0 +1,87 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  display: flex;
+  flex-direction: column;
+}
+
+.card__image {
+  height: 134px;
+  width: 160px;
+  margin: 32px auto 40px;
+}
+
+.card__item-name {
+  line-height: 18px;
+  font-size: 12px;
+  font-weight: 500;
+  margin: 0 16px 4px;
```

Comment:
Will be better to specify correct padding for the whole card, instead of left-right margins for all elements, fix it